Makefile

```diff
@@ -49,9 +56,58 @@ test:
 
 rootfs: out/rootfs.vhd
 
-out/rootfs.vhd: out/rootfs.tar.gz bin/cmd/tar2ext4
+snp: out/kernelinitrd.vmgs out/containerd-shim-runhcs-v1.exe out/rootfs.hash.vhd out/rootfs.vhd out/v2056.vmgs
```

Comment:
Agree containerd-shim doesn't need to be build in this target, I've removed it. However, it's very useful to be able to able to build the release artifacts locally for testing and development - ideally the ADO task would just call make so that the local build is identical to the ADO build. I think it's important to keep targets that allow easy local testing.